src/components/Dashboard/index.jsx

```diff
@@ -217,7 +217,7 @@ export default class Dashboard extends Component {
             {showLogo ? (
               <img
                 className={classes.logoStyle}
-                height={40}
+                height={mobileOpen ? 30 : 40}
```

Comment:
Let's have the logo be the same height for both desktop and mobile.

```jsx
height={30}
```

Reply:
Gotcha! :)